components/project-card/project-card-detailed.scss

```
@@ -0,0 +1,160 @@
+.project-card.detail-view {
+  background: inherit;
+
+  .circle-heart {
+    display: inline-block;
+    width: 37px;
+    height: 37px;
+    background-image: url(/assets/svg/icon-bookmark-entry-detailed.svg);
+    background-position: center center;
+    background-repeat: no-repeat;
+    cursor: pointer;
+
+    &.beating {
+      animation-name: beating;
+      animation-duration: 0.4s;
+      animation-fill-mode: both;
+    }
+  }
+
+  &.bookmarked .circle-heart {
+    background-image: url(/assets/svg/icon-bookmark-selected-entry-detailed.svg);
+  }
+
+  .issues-and-tags-container {
+    @include scaleText(16px, 24px, 20px, 28px);
+
+    color: $dark-blue;
+
+    @media (min-width: $bp-md) {
+      font-weight: 300;
+    }
+  }
+
+  .get-involved {
+    background-color: $gray-05;
+  }
+
+  .description {
+    p {
```

There could be multiple `<p>` within `.description`. I believe we want to preserve the bottom margin except just the last `<p>`.

```suggestion
    p:last-child {
```

---

ooooooo yes, good catch—thank you!